packages/react/src/themes/teams/components/Label/labelVariables.ts

```diff
@@ -9,29 +9,44 @@ export interface LabelVariables {
   startPaddingLeft: string
   endPaddingRight: string
   height: string
-  iconColor: string
+}
+
+export const invertColorScheme = colorScheme => {
```

Comment:
```suggestion
export const invertColorScheme = (colorScheme: ColorScheme) => {
```

Please define input's type